internal/binutils/binutils.go

```diff
@@ -144,7 +144,10 @@ func initTools(b *binrep, config string) {
 		// that if the tool is not found by its default name.
 		b.addr2line, b.addr2lineFound = findExe("gaddr2line", append(paths["addr2line"], defaultPath...))
 	}
-	b.nm, b.nmFound = findExe("nm", append(paths["nm"], defaultPath...))
+	// The "-n" option is supported by llvm since 2011. The output of llvm-nm
+	// and GNU nm with "-n" option is interchangeable for our purposes, so we do
+	// not need to differrentiate them.
+	b.nm, b.nmFound = findNm(append(paths["nm"], defaultPath...))
```

Comment:
It might be good to move the code for locating addr2line above to a separate function findAddr2line as well, to make the code for searching all tools look consistently.

Reply:
Could also see if it makes sense to unify / reuse the code between these two.

```
b.addr2line, b.addr2lineFound = findExe(["addr2line"], ["gaddr2line"], append(paths["addr2line"], defaultPath...))
b.nm, b.nmFound = findExe(["llvm-nm", "nm"], ["gnm"], append(paths["nm"], defaultPath...))
```

where the first argument is the list of names to search on both Linux and OSX and the second is OSX specific.

This also raises a question of whether we should a TODO to support llvm-addr2line?

Reply:
Done. Sent https://github.com/google/pprof/issues/553 for tracking.